.gitignore

```diff
@@ -23,3 +23,4 @@ parts/
 stage/
 *.snap
 *.bz2
+.idea
```

Comment:
Please don't add this (we don't want to maintain a list of various editor state dirs).

Reply:
but then I will certainly commit my files to git at some point because goland asks you all the time

Reply:
@cromefire add your .gitignore to itself so changes aren't tracked.

Reply:
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
Well I and certainly also other people have some files of .idea committed, but wasn't there some hidden ignore thing in .git?

Reply:
Oh yes sorry